Feliz/Styles.fs

```diff
@@ -2194,6 +2194,69 @@ module style =
         /// Inherits this property from its parent element.
         static member inline inheritFromParent = Interop.mkStyle "boxShadow" "inherit"
 
+    [<Erase>]
+    type width =
+        /// Inherits this property from its parent element.
+        static member inline inheritFromParent = Interop.mkStyle "width" "inherit"
+        /// Sets this property to its default value.
+        static member inline initial = Interop.mkStyle "width" "initial"
+        /// Resets this property to its inherited value if it inherits from its parent, and to its initial value if not.
+        static member inline unset = Interop.mkStyle "width" "unset"
+
+        /// The larger of either the intrinsic minimum width or the smaller of the intrinsic preferred width and the available width
+        [<Experimental("This is an experimental API that should not be used in production code.")>]
+        static member inline fitContent = Interop.mkStyle "width" "fit-content"
+
+        /// The intrinsic preferred width.
+        [<Experimental("This is an experimental API that should not be used in production code.")>]
+        static member inline maxContent = Interop.mkStyle "width" "max-content"
+
+        /// The intrinsic minimum width.
+        [<Experimental("This is an experimental API that should not be used in production code.")>]
+        static member inline minContent = Interop.mkStyle "width" "min-content"
+
+    [<Erase>]
+    type minWidth =
+        /// Inherits this property from its parent element.
+        static member inline inheritFromParent = Interop.mkStyle "min-width" "inherit"
```

Comment:
Shouldn't inline styles follow React's conventions? i.e.
```fs
Interop.mkStyle "minWidth" "inherit"
```
instead of
```fs
Interop.mkStyle "min-width" "inherit"
```

Reply:
That's right! Oddly, in `type minHeight` and `type maxHeight` they are specified with `kebab-case` and they work as well 😅

Reply:
I think React will change them into camelCase but will throw warnings in the console when you use it like that